book/release-service.md

```diff
@@ -39,8 +28,95 @@ The diagram below shows the flow of custom resources and the orchestration of pi
 
 ![](../diagrams/release-service/hacbs-release-service-data-flow.jpg)
 
+## Terminology
+
+**Snapshot** -  The custom resource that contains the list of all Components of an Application with their Component Image digests. Once created, the list of Components with their images is immutable. The Integration service updates the status of the resource to reflect the testing outcome.
```

Comment:
You have two spaces instead of 1 after the hyphen

Reply:
done